akka-http-tests/src/test/scala/akka/http/scaladsl/server/DontLeakActorsOnFailingConnectionSpecs.scala

```diff
@@ -6,51 +6,52 @@ package akka.http.scaladsl.server
 
 import java.util.concurrent.{ CountDownLatch, TimeUnit }
 
-import com.typesafe.config.ConfigFactory
-
-import scala.concurrent.Await
-import scala.concurrent.duration._
-import scala.util.{ Failure, Success, Try }
 import akka.actor.ActorSystem
 import akka.event.Logging
 import akka.http.impl.util.WithLogCapturing
 import akka.http.scaladsl.Http
 import akka.http.scaladsl.model.{ HttpRequest, HttpResponse, Uri }
-import akka.http.scaladsl.settings.ConnectionPoolSettings
-import akka.stream.ActorMaterializer
+import akka.stream.{ ActorMaterializer, StreamTcpException }
 import akka.stream.scaladsl.{ Sink, Source }
 import akka.stream.testkit.Utils.assertAllStagesStopped
-import akka.testkit.{ SocketUtil, TestKit }
+import akka.testkit.{ EventFilter, TestKit }
+import com.typesafe.config.ConfigFactory
 import org.scalatest.{ BeforeAndAfterAll, Matchers, WordSpecLike }
 
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import scala.util.{ Failure, Success, Try }
+
 abstract class DontLeakActorsOnFailingConnectionSpecs(poolImplementation: String)
   extends WordSpecLike with Matchers with BeforeAndAfterAll with WithLogCapturing {
 
   val config = ConfigFactory.parseString(s"""
     akka {
+      loggers = ["akka.testkit.TestEventListener"]
       # disable logs (very noisy tests - 100 expected errors)
       loglevel = DEBUG
       akka.loggers = ["akka.http.impl.util.SilenceAllTestEventListener"]
```

Comment:
Woops, that one is broken from my commit.

Reply:
I see now why I thought it was fixed, I'll make it right